core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/dot-binary-settings/dot-binary-settings.component.ts

```diff
@@ -3,16 +3,16 @@ import { forkJoin, of } from 'rxjs';
 import { NgFor } from '@angular/common';
 import { HttpErrorResponse } from '@angular/common/http';
 import {
-    ChangeDetectionStrategy,
-    Component,
-    DestroyRef,
-    EventEmitter,
-    Input,
-    OnChanges,
-    OnInit,
-    Output,
-    SimpleChanges,
-    inject
+  ChangeDetectionStrategy,
+  Component,
+  DestroyRef,
+  EventEmitter,
+  Input,
+  OnChanges,
+  OnInit,
+  Output,
+  SimpleChanges,
+  inject,
```

Comment:
Not sure what happens, but looks like we are in 2 spaces size format now... I am ok with it, but I rather to automate the whole thing in all the project and not per file because that will break code reviews.

@oidacra you know something about it?

Reply:
Now you can. I made a temporary fix.